### ISSUE TITLE
Laravel 5.3

### ISSUE THREAD

**Issue by michaelpittino** [duxet/laravel-rethinkdb#37]
Hello, do you have any plans updating this gem for Laravel 5.3? :)

**Comment by zaherg** [comment]
@michaelpittino any particular issues you got with 5.3 , did it crash or failed to work ?
can you provide more info

**Comment by michaelpittino** [comment]
Yes, I get many compatibility issues. I think some function signatures changed in 5.3

**Comment by zaherg** [comment]
can you provide examples ?

**Comment by duxet** [comment]
I haven't touched 5.3 yet, i will try to fix it later today.

**Comment by dansup** [comment]
I got this working with Laravel 5.3 using the demo examples in the readme, I will make a PR when I update the other functions.

[https://github.com/dansup/laravel-rethinkdb/tree/laravel-53](https://github.com/dansup/laravel-rethinkdb/tree/laravel-53)

**Comment by ghost** [comment]
I'm getting this error when I run composer require "duxet/laravel-rethinkdb:dev-master"

```
PHP Parse error:  syntax error, unexpected '$schedule' (T_VARIABLE) in /var/www/html/rethink/app/Console/Kernel.php on line 40
Script php artisan optimize handling the post-update-cmd event returned with error code 255

Installation failed, reverting ./composer.json to its original content.
```
